File: Kap17/Python/Listing03/FindPrimes.py
   
import time
import math

def getPrimes(maxVal):
    start = time.time()
    primes = [];
                    
    for i in range(2, maxVal, 1):
        isPrime = True;
        for d in range(2, i, 1):
            if i%d == 0:
                isPrime = False
            if isPrime:
                primes.append(i)

    print(math.floor((time.time() - start)*1000))
    return primes

primes = getPrimes(100000)

File: Kap17/PHP/Listing03/FindPrimes.php
<?php
    $primes = getPrimes(100000);
   
    function getPrimes($maxVal) {
        $start = floor(microtime(true) * 1000);
        $primes = array();
                    
        for ($i = 2; $i <= $maxVal; $i++) {
            $isPrime = TRUE;
            for ($d = 2; $d < $i; $d++) {
                if ($i%$d == 0) {
                    $isPrime = FALSE;
                }
            } 
            if ($isPrime) {
                array_push($primes, $i);
            }
        }

        echo (floor(microtime(true) * 1000) - $start);
        return $primes;
    }

?>